src/create_disk.sh

```diff
@@ -544,16 +551,50 @@ create_dmverity() {
     mount -o ro "/dev/mapper/${partlabel}" "${mountpoint}"
 }
 
+# Save genprotimg input for later and don't run zipl here
+rdcore_replacement() {
+    local se_script_dir="/usr/lib/coreos-assembler/secex-genprotimgvm-scripts"
+    se_tmp_disk=$(realpath /dev/disk/by-id/virtio-genprotimg)
+    se_tmp_mount=$(mktemp -d -p "/tmp" genprotimg-XXXXXX)
+    se_tmp_boot="${se_tmp_mount}/genprotimg"
+    mount "${se_tmp_disk}" "${se_tmp_mount}"
+    mkdir "${se_tmp_boot}"
+
+    se_initrd="${se_tmp_boot}/initrd.img"
+    se_kernel="${se_tmp_boot}/vmlinuz"
+    se_parmfile="${se_tmp_boot}/parmfile"
+
+    blsfile=$(find "${rootfs}"/boot/loader/entries/*.conf)
```

Comment:
I tested it and `grep` is lazy and does not do any pattern matching, but instead just tries to use the path as is...
I think overall it is much cleaner to have the variable be the actual file, instead of a pattern that needs to be matched each time it is used.

Reply:
Sorry, my mistake.  I thought the wildcard would be evaluated at variable assignment, not at command invocation.